docs/packaging_policy.md

```diff
@@ -108,29 +108,54 @@ for each combination. There are some particular cases for this general rule:
 ## Options
 
 Recipes can list any number of options with any meaning, and defaults are up to the recipe itself. The CI cannot enforce anything
-in this direction. However, there are a couple of options that have a special meaning for the CI:
+in this direction. However, there are a couple of options that have a special meaning for the CI.
 
-* `shared` (with values `True` or `False`). The CI inspects the recipe looking for this option. If it is found, it will
+### Predeinfed Options and Known Defaults
+
+ConanCenter supports many combinations, these are outline in the [supported configurations](supported_platforms_and_configurations.md) document for each platform.
+By default, `shared=False` is compiled with `fPIC=True`. If support, `header_only=False` is the default.
```

Comment:
I'm not sure if I'm getting the meaning of this line.

Reply:
```suggestion
By default recipes should use `shared=False` with `fPIC=True`. If support, `header_only=False` is the default.
```